backend/conduit/tags/views.py

```diff
@@ -85,13 +86,50 @@ def get_members_from_tag(slug):
         res['followers'].append(follower['profile'])
     return res
 
-@blueprint.route('/api/tags/<slug>/moderator', methods=('POST',))
+@blueprint.route('/api/tags/<slug>/admin', methods=('POST',))
 @jwt_required
-@marshal_with(tag_schema)
 def claim_tag(slug):
     if not current_user.isAdmin:
         raise InvalidUsage.not_admin()
     tag = Tags.query.filter_by(slug=slug).first()
-    tag.addModerator()
+    tag.addModerator(current_user.profile)
     tag.save()
-    return tag
+    moderators = profile_schemas.dump(tag.moderators)
+    response = {
```

Comment:
Instead of serializing the data by yourself, you should use marshmallow to do the work for you. Thinking about it now i think it would make more sense to return all the information for a tag.